res/blink.js

```diff
@@ -11,12 +11,10 @@
   var sock = new WebSocket(ws);
 
   function msg(t, m) {
-    if (m === undefined) {
-      m = t;
-    } else {
-      m.type = t;
-    }
-    sock.send(JSON.stringify(m));
+    sock.send(JSON.stringify({
+      type: m === undefined ? t.type : t,
+      data: m
```

Comment:
I think the idea of this was to support both `msg('foo', {bar: 'baz'})` and `msg({type: 'foo', bar: 'baz'})`; this would behave differently in the second case. If we make this change we should probably just get rid of that altogether.